### PR TITLE
System Check Script #3: I2C Scan

### PR DESCRIPTION
**Pull request by yashika-batra**
I2C scan returns the addresses of devices connected to each I2C bus.

### CODE REVIEW

File: state_machine/applications/system-check/tests/i2c_scan.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    cubesat.i2c2.unlock()
+
+    while not cubesat.i2c3.try_lock():
+        pass
+    print("Addresses of Devices Connected to I2C3:",
+          [hex(x) for x in cubesat.i2c3.scan()])
+    cubesat.i2c3.unlock()
+
+    print("I2C Scan Complete.")
+    print("")
```
Comment (thetazero) [inline state_machine/applications/system-check/tests/i2c_scan.py:28]: Not sure why we have this.

**Review by thetazero — CHANGES_REQUESTED** [review]
Looks great!

Just confused about the `print('')`

File: state_machine/applications/system-check/tests/i2c_scan.py (hunk 1 above)
Comment (yashika-batra) [reply state_machine/applications/system-check/tests/i2c_scan.py:28]: This is just included to format the user output (I could've just added a "\n" though to be honest, not sure why I decided to do it this way)

**Review by thetazero — APPROVED** [review]
Looks good

**Pull request merged**